<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calibration_widget</class>
 <widget class="QWidget" name="Calibration_widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="Displayer">
       <property name="minimumSize">
        <size>
         <width>671</width>
         <height>376</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="calibrationDisplayerMode">
       <item>
        <widget class="QPushButton" name="originalImage">
         <property name="text">
          <string>original</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chesscorners">
         <property name="text">
          <string>Chessboard corners</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="undistortedButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="text">
          <string>undistorted image</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="newImageSet">
         <property name="text">
          <string>new image set</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="calibrationOption">
       <item>
        <widget class="QPushButton" name="calibration">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>calibrate</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QCheckBox" name="CharucoCalib">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Charuco calibration</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadCalib">
         <property name="text">
          <string>load calib</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save">
         <property name="text">
          <string>save and quit calibration</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignBottom">
    <widget class="QGroupBox" name="horizontalGroupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="prevImage">
        <property name="text">
         <string>previous image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="nextImage">
        <property name="text">
         <string>next image</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionShow_chess_corners">
   <property name="text">
    <string>show_chess_corners</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
